### DISCUSSION TITLE
Assign an icon to several notes at the same time

### DISCUSSION THREAD

**Discussion by collector-ynh** [TriliumNext/Trilium#7361] [Q&A] [answered]
### Trilium Version

0.58

### What operating system are you using?

Ubuntu

### What is your setup?

Local (no sync)

### Operating System Version

Linux-Mint 21.1

### Description

Hello,
I know that it is possible to select several notes with the "SHIFT" key, and assign them a label or other elements, but I have not found a way to assign an icon to several notes at the same time!

Is it possible to assign an icon to several notes at the same time?

**Comment by zadam** [comment, marked as answer]
So, icon is actually a label, so you can use the same "bulk assign label" mechanism to change icons of many notes. "Full search" has similar capability through "search actions". It's a bit awkward to use for this purpose but could save a lot of time if you have a lot of notes to change.

![image](https://user-images.githubusercontent.com/617641/220770777-0b19dc8a-057e-4a7c-ac06-2d0752883625.png)

**Comment by collector-ynh** [comment]
Thank you. I'll go with it for now, although I would have preferred a graphical feature, I think this will help me a lot!
